src/core/domain/entities/item.py

```diff
@@ -28,3 +29,7 @@ class Item(BaseModel):
     assigned_user_id: Optional[int]
     group_item_id: Optional[int]
     group_status: Optional[int]
+
+
+class ItemFactory(ModelFactory):
```

Comment:
The factories should be created in the `factories.py` file, and this file should be stored under the `tests` directory, because it's part of the  *tests*

Reply:
oh right, that's much cleaner. Done!